File: Python/3_Insertion Sort/Insertion Sort.py
def insertion_sort(arr):
    for k in range(1, len(arr)):
        j = k
        while j > 0 and arr[j-1] > arr[j]:
            arr[j], arr[j-1] = arr[j-1], arr[j]
            j = j - 1

arr = [12, 11, 13, 5, 6]
insertion_sort(arr)
print(arr)

File: Swift/3_Insertion Sort/Insertion Sort.swift
func insertionSort(arr: [Int]) -> [Int] {
    var array = arr
    for k in 1..<array.count {
		var j = k
		while j > 0 && array[j-1] > array[j] {
			let temp = array[j]
			array[j] = array[j-1]
			array[j-1] = temp
			j -= 1
		}
	}
    return array
}

let unsortedArray = [14, 46, 43, 27, 57, 41, 45, 21, 70]
print(insertionSort(arr: unsortedArray))